llama_stack/providers/remote/inference/bedrock/bedrock.py

```diff
@@ -87,9 +87,9 @@ def _to_inference_profile_id(model_id: str, region: str = None) -> str:
 
 class BedrockInferenceAdapter(
     ModelRegistryHelper,
-    Inference,
     OpenAIChatCompletionToLlamaStackMixin,
     OpenAICompletionToLlamaStackMixin,
+    Inference,
```

Comment:
wow this is pretty subtle. can you show the stack trace and which method override caused the issue?